content/api/local-installation.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Local installation
+---
+
+Installing our code locally will allow your editor to extract type definitions from the `Robot` class and its boards, as well as allowing auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
+
+Installing these definitions is highly optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
+
+## Installation
+Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. On Windows, this is best done using the [`get-pip.py` script](https://pip.pypa.io/en/stable/installing).
```

Comment:
Are there any recent versions of Python for Windows which don't bundle pip? (If not, I suggest dropping the last sentence here).